components/brave_wallet_ui/stories/wallet-extension-panels.tsx

```diff
@@ -507,6 +508,10 @@ export const _ConnectedPanel = (args: { locked: boolean }) => {
     console.log(selectedTransaction)
   }
 
+  const onSelectBuyOption = (option: BraveWallet.OnRampProvider) => {
```

Comment:
This function is unnecessary.

Reply:
@josheleonard I have remove the function